src/main/java/org/folio/circulation/resources/renewal/OverrideRenewalStrategy.java

```diff
@@ -77,7 +76,7 @@
         return processRenewal(proposedDueDateResult, loan, comment);
       }
 
-      if (loan.hasItemWithStatus(ItemStatus.DECLARED_LOST)) {
+      if (isLoanLost(loan)) {
```

Comment:
This reads strangely to me. For me, the loan isn't lost, the item is lost whilst on loan to the borrower.

```suggestion
      if (isItemLost(loan)) {
```

Reply:
Renamed.